boards/sparc/qemu_leon3/qemu_leon3.yaml

```diff
@@ -12,3 +12,6 @@ supported:
   - netif
 testing:
   default: true
+  ignore_tags:
+    - net
```

Comment:
According to my understanding, `qemu_leon3` is the only big-endian platform exercised in CI. It could be of interest for the network subsystem since it has previously revealed issues related to endian: 5dcdf198, fb18a6e2, 634b8fd5, 3658f74d. FYI @jukkar